xdsl/ir.py

```diff
@@ -389,11 +392,17 @@ def print_parameters(self, printer: Printer) -> None:
         """Print the attribute parameters."""
         printer.print_paramattr_parameters(self.parameters)
 
+    def _verify(self):
+        # Verifier generated by irdl_attr_def
+        attr_def = type(self).irdl_definition()
+        if attr_def is not None:
+            attr_def.verify(self)
+        super()._verify()
+
     @classmethod
-    @property
```

Comment:
Is there a reason to change that?
I felt that this was okay?

Reply:
Ok I can split this out into a separate PR.